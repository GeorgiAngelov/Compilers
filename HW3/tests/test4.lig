function main(myargs : [[int]]) : int {
      var myargs : int = 0;
      return (myargs);
}
