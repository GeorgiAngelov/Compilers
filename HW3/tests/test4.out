(decls
   (decl-fun:NONE (id main) (type-fun (params (typed-id (id myargs) (type-array (type-array (type-int))))) (type-int))
      (decls
         (decl-var:NONE (id myargs) (type-int) (num:int 0)))
      (stmts
         (return:NONE (id:[[int]] myargs)))))
Typechecking failed!
